index.js

```diff
@@ -544,7 +552,16 @@ module.exports = async function(options) {
                     prefix: '/shared-assets',
                     uploadsPath: getRootDir() + '/sites/public/uploads',
                     uploadsUrl: '/uploads',
-                    tempPath: getRootDir() + '/sites/data/temp/shared-assets/uploadfs'
+                    tempPath: getRootDir() + '/sites/data/temp/shared-assets/uploadfs',
+                    ...(process.env.CDN
+                      ? {
+                        cdn: {
+                          enabled: true,
+                          // uploadfs does not apply the prefix to the CDN unless told to
+                          url: process.env.CDN + '/shared-assets'
```

Comment:
Start out with the right prefix for assets, we'll override as seen elsewhere for attachments